cfme/tests/cloud_infra_common/test_events.py

```diff
@@ -18,6 +17,7 @@
     pytest.mark.provider([CloudProvider, InfraProvider],
```

Comment:
I think you should update this to create an excluded ProviderFilter, and combine it with the current ProviderFilter that you have defined here as kwargs to mark.provider.